client/src/utils/KBar.tsx

```diff
@@ -63,6 +63,120 @@ export const KBar: React.FunctionComponent<PropsWithChildren> = ({ children }) =
         dispatch(fetchAllCharacters())
       },
     },
+    {
```

Comment:
theme color is already configured below I think

Reply:
Same one.. I didn't add it again